packages/app/src/components/prague/GuideDrawer.tsx

```diff
@@ -45,249 +33,114 @@ const Guide = () => {
-                            )}
-                          </ListItem>
-                          <ListItem mb={4}>
-                            {t(
-                              "Tap 'Connect' and follow the instructions to successfully connect your Wallet, (WalletConnect is also available)."
-                            )}
-                          </ListItem>
-                          <ListItem>
-                            {t(
-                              "Once your wallet is connected, enter your MACI key in the field below and press 'Save'."
-                            )}
-                          </ListItem>
-                        </UnorderedList>
-                      </Text>
-                    </VStack>
-                  </AccordionPanel>
-                </AccordionItem>
-              </ListItem>
-            </UnorderedList>
+            <AccordionItem border="none">
+              <VStack>
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 1: Download a wallet that supports Gnosis Chain (xDAI)
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4}>
+                <VStack spacing={6} alignItems="flex-start">
+                  <iframe title="guide" src="https://scribehow.com/page-embed/Mobile_Wallet_Set_up__OZiM3PehSGmXB73cXh85KQ" width="315" height="640" allowFullScreen frameBorder="0"></iframe>
+                </VStack>
+              </AccordionPanel>
+            </AccordionItem>
+          </Accordion>
+          <Accordion allowToggle>
+            <AccordionItem border="none">
+              <VStack>
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 2A: Get some xDAI on Gnosis Chain (xDAI) by using a bridge such as xDAI Bridge
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4}>
+                <VStack spacing={6} alignItems="flex-start">
+                  <iframe title="onramp" src="https://scribehow.com/page-embed/Voters_guide__Desktop__08cXvIydQrWdpcHMfhSuDQ" width="315" height="640" allowFullScreen frameBorder="0"></iframe>
+                </VStack>
+              </AccordionPanel>
+            </AccordionItem>
+          </Accordion>
+          <Accordion allowToggle>
+            <AccordionItem border="none">
+              <VStack>
+                <Text px={0} textAlign={"left"} as={AccordionButton} fontSize="md">
+                  Step 2A: OR get free funds from the xDAI Faucet
+                  <AccordionIcon></AccordionIcon>
+                </Text>
+              </VStack>
+              <AccordionPanel pb={4}>
+                <VStack textAlign={"left"} alignItems={"flex-start"}>
+                  <Text>
+                    Copy the following JSON to your clipboard and paste it into the "ABI" field of the{" "}
```

Comment:
Where is the ABI field? 🧐 

Any reason to go this route vs. having people use the form (i.e. input their address & select an amount? These feel like overly technical instructions for users.

Reply:
In any case, my understanding is that 0.01 xDAI is not enough for them to vote, or is it?

Reply:
Ah, maybe in most cases it is? Looking at previous votes, only 1 cost more: https://gnosisscan.io/address/0xc7b53f0243201425d8bf83f98b9dc43a878bfd20

I'm guessing it varies by the number of projects voted for?

Reply:
it is enough to vote for ~4 projects